packages/terra-clinical-item-collection/tests/jest/__snapshots__/Item.test.jsx.snap

```diff
@@ -11,12 +11,8 @@ exports[`List View Tests should pass customProps to the Item 1`] = `
     accessoryAlignment="alignCenter"
     displays={
       Array [
-        <ItemDisplay
-          iconAlignment="center"
-          isDisabled={false}
-          isTruncated={false}
+        <InjectIntl(ItemDisplay)
           text="Display 1"
-          textStyle="primary"
         />,
       ]
     }
```

Comment:
These tests will need updating as these snapshots are now too high level due to the addition of internationalization. However, it is now outside the scope of this PR so we can make a follow-up Jira for it.

Reply:
Logged UXPLATFORM-9008